src/components/WordGroupSelection/WordGroupSelection.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
+import {collectedWordGroupsService} from "util/GWUtil/resource";
+import "./WordGroupSelection.css";
+import WordGroupIcon from "./WordGroupIcon";
+import WordSelection from "./WordSelection";
+
+function WordGroupSelection(props) {
+    var word_groups = null;
+    var word_keys = null;
+    const [peopleWords, setPeopleWords] = useState(null);
+    const [actionWords, setActionWords] = useState(null);
+    const [toyWords, setToyWords] = useState(null);
+    const [colorWords, setColorWords] = useState(null);
+    const [animalWords, setAnimalWords] = useState(null);
+    const [transportWords, setTransportWords] = useState(null);
+    const [bodyWords, setBodyWords] = useState(null);
+    const [clothingWords, setClothingWords] = useState(null);
+    const [foodWords, setFoodWords] = useState(null);
+    const [foodWords2, setFoodWords2] = useState(null);
+    const [furnitureWords, setFurnitureWords] = useState(null);
+    const [emotionWords, setEmotionWords] = useState(null);
+    const [mediaWords, setMediaWords] = useState(null);
+    const [animalWords2, setAnimalWords2] = useState(null);
+    const [animalWords3, setAnimalWords3] = useState(null);
```

Comment:
maybe instead of having a separate state variable for each word, you could have an object with keys that are the names of the groups and values that are the arrays of words?